core/openapi/schemas/party.yaml

```diff
@@ -1,103 +1,89 @@
-                description: |-
-                  The key name indicates the type of address (e.g. office,
-                  home, etc.). The value is the address itself.
-                properties:
-                  deliveryPoint:
-                    type: string
-                    description: Address line for the location.
-                  city:
-                    type: string
-                    description: City for the location.
-                  administrativeArea:
-                    type: string
-                    description: State or province of the location.
-                  postalCode:
-                    type: string
-                    description: ZIP or other postal code.
-                  country:
-                    type: string
-                    description: Country of the physical address.
-          url:
-            $ref: common/link.yaml
-            description: On-line information about the responsible party.
-          hoursOfService:
+        description: Telephone numbers at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Time period (including time zone) when the resposible party
-              can be contacted.
-          contactInstructions:
+              The key name indicates the type of phone number (e.g. home,
+              work, fax, etc.). The value is the phone number itself.
+            pattern: "^+[1-9]{1}[0-9]{3,14}$"
+      email:
+        type: object
+        description: Email address at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Supplemental instructions on how or when to contact the
-              responsible party.
-      roles:
-        $ref: roles.yaml
+              The key name indicates the type of email address (e.g. home,
+              work, etc.). The value of the email address itself.
+            format: email
+      address:
+        type: object
+        description: Physical location at which contact can be made.
+        patternProperties:
+          "^.*":
+            type: object
+            description: |-
+              The key name indicates the type of address (e.g. office,
+              home, etc.). The value is the address itself.
+            properties:
+              deliveryPoint:
+                type: string
+                description: Address line for the location.
+              city:
+                type: string
+                description: City for the location.
+              administrativeArea:
+                type: string
+                description: State or province of the location.
+              postalCode:
+                type: string
+                description: ZIP or other postal code.
+              country:
+                type: string
+                description: Country of the physical address.
+      url:
```

Comment:
I think the name "url" is a bit misleading. This is actually an object, not just a url. Maybe rename to link or so? ISO calls it onlineResource (but I don't like that).

Reply:
As with logo, which relation type should be used? I assume there are different use cases (e.g. homepage of company vs. vCard), but a bit of guidance would be appreciated. It's required so people will stumble across it (as I do right now when creating an example file).

Reply:
@m-mohr change the name of `url` to links and made it an array.  Also added requirement that all links in the `links` list shall use the rel `about` and include a media type indicating the content type of the link (e.g. text/html for a company's web page versus text/vcard for virtual contact file).